index.bs

```diff
@@ -169,8 +169,9 @@ When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>,
   - The content of an [=AV1 Image Item=] is called the <dfn>AV1 Image Item Data</dfn> and shall obey the following constraints:
     - The [=AV1 Image Item Data=] shall be identical to the content of an [=AV1 Sample=] marked as 'sync', as defined in [[!AV1-ISOBMFF]].
     - The [=AV1 Image Item Data=] shall have exactly one [=Sequence Header OBU=].
-    - The [=AV1 Image Item Data=] should have its <code>[=still_picture=]</code> flag set to 1.
-    - The [=AV1 Image Item Data=] should have its <code>[=reduced_still_picture_header=]</code> flag set to 1.
+    - If the [=AV1 Image Item Data=] is made of a single frame (i.e. when using a single layer),
```

Comment:
Nit: "made up of" sounds better than "made of" here.

Reply:
Or "consists of"?

Reply:
Yes, I like "consists of" better.